app/_hub/kong-inc/correlation-id/1.0.0.md

```diff
@@ -0,0 +1,127 @@
+---
+name: Correlation ID
+publisher: Kong Inc.
+version: 1.0.0
```

Comment:
```suggestion
version: 1.0.x
```
File needs to be renamed to 1.0.x and a 2.2.x version needs to be added to the yaml file. @locao what changed for 2.2? Just supported by release?

Reply:
I noticed that this plugin had its version bumped from 1.0 to 2.0 a long time ago (May 2019) but we didn't update the docs accordingly. There are no other relevant changes we should add to its docs.